pkg/objstore/objstore.go

```diff
@@ -83,6 +83,14 @@ type InstrumentedBucketReader interface {
 	ReaderWithExpectedErrs(IsOpFailureExpectedFunc) BucketReader
 }
 
+type ObjectAttributes struct {
+	// Size is the object size in bytes.
+	Size int64 `json:"size"`
```

Comment:
```suggestion
	Size uint64 `json:"size"`
```

Reply:
I've intentionally used `int64` because all objstore clients use that data type for the size, so using `uint64` looks a bit unnatural. What do you think?

Reply:
It's ok to me